scripts/build_lib.sh

```diff
@@ -19,6 +19,22 @@ toggle_failpoints() {
   mode="$1"
   if command -v gofail >/dev/null 2>&1; then
     run gofail "$mode" server/etcdserver/ server/storage/backend/
+    (
+      cd ./server
+      run go get go.etcd.io/gofail/runtime
+    ) || exit 2
+    (
+      cd ./etcdutl
+      run go get go.etcd.io/gofail/runtime
+    ) || exit 2
+    (
+      cd ./etcdctl
+      run go get go.etcd.io/gofail/runtime
```

Comment:
It isn't needed to get `gofail/runtime` under `etcdctl`.

Reply:
etcdctl import server

Reply:
ack

Reply:
Based on my previous test, we don't need to get `gofail/runtime` under `etcdctl`. 

But it isn't a big deal.